shared/js/initApp.js

```diff
@@ -302,8 +302,11 @@ if (appOptions.droplet) {
   promise = loadSource('jsinterpreter/acorn_interpreter')()
       .then(loadSource('requirejs/require'))
       .then(loadSource('ace/ace'))
+      .then(loadSource('ace/mode-javascript'))
```

Comment:
Preload mode-javascript, this way we can safely ace require it and the ace modules it uses.